csrc/cutlass_extensions/vllm_numeric_conversion.cuh

```diff
@@ -0,0 +1,538 @@
+// Based off of: cutlass/numeric_conversion.h
+
+#pragma once
+
+#include "cutlass/numeric_conversion.h"
+#include "cutlass_extensions/vllm_custom_types.cuh"
+
+namespace cutlass {
+
+// TODO (Lucas): Implement
+// for Array<cutlass::float8_e4m3fn, N> <= Array<vllm_uint4b8_t, N>
+
+// ....
+
+// for Array<cutlass::half_t, N> <= Array<vllm_uint4b8_t, N>
+template <FloatRoundStyle Round, int N>
+struct NumericArrayConverter<cutlass::half_t, vllm_uint4b8_t, N, Round> {
+  using result_type = Array<cutlass::half_t, N>;
+  using source_type = Array<vllm_uint4b8_t, N>;
+
+  static FloatRoundStyle const round_style = Round;
+
+ private:
+  using result_type_packed_8 = Array<cutlass::half_t, 8>;
+  using result_type_packed_4 = Array<cutlass::half_t, 4>;
+  using result_type_packed_2 = Array<cutlass::half_t, 2>;
+  using source_type_packed_8 = Array<vllm_uint4b8_t, 8>;
+  using source_type_packed_4 = Array<vllm_uint4b8_t, 4>;
+  using source_type_packed_2 = Array<vllm_uint4b8_t, 2>;
+
+  // Not Valid, not supported
+  using ScalarConverter =
+      NumericConverter<cutlass::half_t, vllm_uint4b8_t, Round>;
```

Comment:
What do you mean by this comment?

Reply:
just that `ScalarConverter` is required but `NumericConverter<cutlass::half_t, vllm_uint4b8_t, Round>` doesn't actually work, its just there so it compiles, ill make the comment more clear